keps/sig-security/3203-auto-refreshing-official-cve-feed/README.md

```diff
@@ -0,0 +1,478 @@
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for
+  publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to
+  mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+Currently it is not possible to filter for issues or PRs that are related to
+CVEs announced by kubernetes. This KEP addresses this concern by labelling this
+issues or PRs with the new label **official-cve-feed** using the automation. The
+in-scope issues are the closed issues for which there is a CVE ID and is
+officially announced as a Kubernetes CVE by SRC in the past.
+
+## Motivation
+
+With the growing number of eyes on Kubernetes, the number of CVEs related to
+Kubernetes have increased. Although most CVEs are regularly fixed that directly
+or indirectly or transitively impact Kubernetes, there is no single place to
+programmatically subscribe or pull the data of fixed CVEs, for the end users of
+Kubernetes. Current options are either broken or incomplete.
+
+An auto-refreshing CVE feed will allow end users to programmatically fetch the
+list of CVEs and allow them to get the latest information from kubernetes
+community.
+
+### Goals
+
+Create a periodically auto-refreshing machine-readable list of official
+Kubernetes CVEs
+
+### Non-Goals
+
+- Triage and vulnerability disclosure: This will continue to be done by SRC
+- Listing CVEs that are identified in build time dependencies and container
+  images. Only official CVEs announced by the Kubernetes SRC will be published
+  in the feed.
+- Integration with [CVEProject](https://github.com/CVEProject/cvelist) may
+  happen at a future stage but currently is not planned or scoped.
+
+### User Stories (Optional)
+
+#### Story 1
+
+As a K8s end user, I want a list of CVEs with relevant information that I can
+fetch programmatically, so I can understand when new CVEs are announced
+
+#### Story 2
+
+As a K8s End User, I want to use my browser to get a list of fixed CVEs, from
+the official K8s website so that I can trust it as an authoritative source of
+data through implicit trust offered by website certificate and domain name.
+
+#### Story 3
+
+As a K8s maintainer, I want to create a process that auto-updates CVE feed, when
+SRC announces new CVEs such that I do not have to do extra work to maintain this
+feed manually
+
```

Comment:
#### Story 4

As a K8s platform provider maintainer, I want to automatically to know if my kubernetes clusters are vulnerable to any of the CVE:s SRC have announced. I want to have a programmatically available API to parse this kind of data so I can easily provide it in to to my platform.

Reply:
Added